mkdocs/docs/experiments/categories/aws-ssm/AWS-SSM-experiments-tunables.md

```diff
@@ -6,7 +6,8 @@ It stressed the `CPU_CORE` cpu cores of the `EC2_INSTANCE_ID` ec2 instance and `
 
 Use the following example to tune this:
 
-[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/aws-ssm/common/cpu-cores.yaml yaml)
+[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/aws-ssm/common/cpu-cores.yaml yaml"
```

Comment:
Some of these changes will break the docs. It's probably due to your IDE auto formatting, revert the commits and disable your ide formatting first and then run replace. And make sure to only run replace on the env vars and not on any other texts.

Reply:
On it.